src/operations/update_by_id.test.ts

```diff
@@ -606,32 +606,6 @@ describe('mapAccessPatterns', () => {
     ).toThrow(InvalidIndexedFieldValueException);
   });
 
-  it.each([null, undefined, ''])(
```

Comment:
We should have a default handling for `null` values in an index - ideally when we convert them they become empty strings. I'd like to see a test for this

Reply:
yep, there is a test [here](https://github.com/gberdzenishvili/dynaglue/blob/05a2412483791a14688961bf1a7c6a4c5ba2ab4c/src/operations/update_by_id.test.ts#L875) and e2e test [here](https://github.com/gberdzenishvili/dynaglue/blob/6bf45b73caffbdec94d1a17e8f285a7f73d04f0f/src/operations/e2e.test.ts#L275)